portal/templates/profile_macros.html

```diff
@@ -576,7 +578,7 @@ <h4 class="text-success registration-label">registered</h4>
                         $(this).prop("checked", true);
                     };
                 });
-            {% endif %}
+            {%- endif -%}
```

Comment:
Paul, the '-' is to strip the unnecessary whitespace.  I see the if statement rendered as a blank line so decide to strip the whitespace it created.